app/views/layouts/playbook/_sidebar.html.erb

```diff
@@ -1,25 +1,28 @@
 <br>
-<%= pb_rails("nav", props: {title: "Kits", link: kits_path, variant: "subtle"}) do %>
+<% all_link = @type == 'react' ? kits_path('', type: 'react') : kits_path %>
```

Comment:
The left-hand assignments in this template make me feel uneasy. Is there another place that we could potentially place these?

Reply:
I could move this to a helper